pkg/gopass/secrets/kv.go

```diff
@@ -102,28 +104,49 @@ func (k *KV) Keys() []string {
 	for key := range k.data {
 		keys = append(keys, key)
 	}
+	sort.Strings(keys)
 	if _, found := k.data["password"]; !found {
-		keys = append(keys, "password")
+		keys = append([]string{"password"}, keys...)
 	}
-	sort.Strings(keys)
 	return keys
 }
 
-// Get returns a single key
+// Get returns the first value of that key
 func (k *KV) Get(key string) (string, bool) {
 	key = strings.ToLower(key)
+
+	if v, found := k.data[key]; found {
+		return v[0], true
```

Comment:
this is deterministic, isn't it?
the values should be in that slice in the order they are defined in the secret?

Reply:
Yes, they should